src/components/SpaceProposalVotesModal.vue

```diff
@@ -0,0 +1,238 @@
+import {
+  ExtendedSpace,
+  Proposal,
+  Vote,
+  SpaceStrategy
+} from '@/helpers/interfaces';
+import {
+  useProfiles,
+  useWeb3,
+  useIntl,
+  useInfiniteLoader
+} from '@/composables';
+import { getProposalVotes } from '@/helpers/snapshot';
+
+const props = defineProps<{
+  space: ExtendedSpace;
+  proposal: Proposal;
+  votes: Vote[];
+  loaded: boolean;
+  strategies: SpaceStrategy[];
+  userVote: Vote | null;
+  loadingMore: boolean;
+  open: boolean;
+}>();
+
+defineEmits(['loadVotes', 'close']);
+
+const { formatCompactNumber } = useIntl();
+const { web3Account } = useWeb3();
+
+const votes = ref<Vote[]>([]);
+const votesScrollWrapper = ref<HTMLElement | null>(null);
+const authorIpfsHash = ref('');
+const modalReceiptOpen = ref(false);
+const voteQuery = ref('');
+
+const voteCount = computed(() => props.proposal.votes);
+
+const sortedVotes = computed(() => {
+  const votesClone = clone(votes.value);
+  if (props.userVote) votesClone.push(props.userVote);
+  const uniqVotes = uniqBy(votesClone, 'ipfs' as any);
+  if (uniqVotes.map(vote => vote.voter).includes(web3Account.value)) {
+    uniqVotes.unshift(
+      uniqVotes.splice(
+        uniqVotes.findIndex(item => item.voter === web3Account.value),
+        1
+      )[0]
+    );
+  } else {
+    uniqVotes.sort((a, b) => b.balance - a.balance);
+  }
+  return uniqVotes;
+});
+
+const titles = computed(() =>
+  props.strategies.map(strategy => strategy.params.symbol || '')
+);
+
+function isZero() {
+  if (!props.loaded) return true;
+  if (votes.value.length > 0) return true;
+}
+
+function openReceiptModal(iphsHash) {
+  authorIpfsHash.value = iphsHash;
+  modalReceiptOpen.value = true;
+}
+
+const { profiles, loadProfiles } = useProfiles();
+
+watch(sortedVotes, () => {
+  loadProfiles(sortedVotes.value.map(vote => vote.voter));
+});
+
+const { loadBy, loadingMore, loadMore } = useInfiniteLoader(10);
+
+function formatProposalVotes(votes) {
+  if (!votes.length) return [];
+  return votes.map(vote => {
+    vote.balance = vote.vp;
+    vote.scores = vote.vp_by_strategy;
+    return vote;
+  });
+}
+
+async function loadMoreVotes() {
+  const votesObj = await getProposalVotes(props.proposal.id, {
+    first: loadBy,
+    skip: votes.value.length
+  });
+  votes.value = votes.value.concat(formatProposalVotes(votesObj));
+}
+
+const { arrivedState } = useScroll(votesScrollWrapper, {
+  throttle: 500,
+  offset: {
+    bottom: 300
+  }
+});
+
+const { bottom } = toRefs(arrivedState);
+
+watch(
+  () => [props.open, bottom.value],
+  to => {
+    if (props.open && to) {
+      loadMore(loadMoreVotes);
+    }
+  },
+  {
+    immediate: true
+  }
+);
+
+watch(
+  () => props.votes,
+  to => {
+    if (to.length > 0) {
+      votes.value = [...to];
+    }
+  }
+);
+
+watch(
+  () => props.open,
+  to => {
+    if (!to) {
+      votes.value = [...props.votes];
+    }
+  }
+);
+</script>
+
+<template>
+  <BaseModal
+    :open="open"
+    max-height="500px"
+    max-width="700px"
```

Comment:
@bonustrack could you comment on design. I'm pretty sure we don't want to change the modal width correct?

Also we want to keep the search and modal aligned like everywhere else I think 
<img width="533" alt="image" src="https://user-images.githubusercontent.com/51686767/225286970-e5d0faee-571e-4d9a-a710-a85e51c5a877.png">

Reply:
I changed modal maxWidth to preserve Votes block with from Proposal page to make it the same.

Reply:
Yeah, lets see what designer Fab has to say about it.

Reply:
The modal should look like others modal, with same width. Also on small screen it should take the whole height, currently it's cut:
<img width="520" alt="image" src="https://user-images.githubusercontent.com/16245250/226104297-67a156c0-f3dc-4c2a-8f7d-965471a3bf19.png">